components/ImagePopover/index.js

```diff
@@ -0,0 +1,79 @@
+/** @flow */
+
+require('./styles.css');
+
+import React from 'react';
+var {PropTypes} = React;
+
+//todo: handle configuring desired placement with prop
+//todo: logic to ensure that edge-item popovers don't pop out of the viewport
+
+class ImagePopover extends React.Component {
+  onMouseEnter(e) {
+    e.preventDefault();
+    this.props.handleMouseEnter && this.props.handleMouseEnter(this.props.data)
+  }
+
+  onMouseLeave(e) {
+    e.preventDefault();
+    this.props.handleMouseLeave && this.props.handleMouseLeave(this.props.data)
+  }
```

Comment:
From my previous chats with Mel, this effect should *only* take place on a `click` or `tap`-style event.